README.md

```diff
@@ -6,41 +6,126 @@ Transform FHIR to dataset for ML applications
 
 This project is still under development.
 
-This repo is a POC allowing to make a query (close to SQL format) on a FHIR API and to retrieve tabular data.
+This repo is a POC allowing to make a SQL query on a FHIR API and to retrieve tabular data. 
 
-The request that FHIR2Dataset performs on a FHIR API is specified in a configuration file of this [form](examples/config_template.json).
-The purpose of this file is to perform a query whose result will be identical to the next SQL query:
+## Installation
+
```

Comment:
As we aim to publish the new version on PyPI, we could be announced that it's possible to install it with `pip`!

```suggestion
### With pip

```
pip install fhir2dataset
```

### From source
```